static/i18n/en/app.json

```diff
@@ -867,7 +872,7 @@
   "DeviceAction": {
     "allowAppPermission": "Open the {{wording}} app on your device",
     "allowAppPermissionSubtitleToken": "to manage your {{token}} tokens",
-    "allowManagerPermission": "{{wording}} on your device",
+    "allowManagerPermission": "Allow Ledger Manager on your device",
```

Comment:
we never should have "Ledger Manager" in the wording, the goal is always to have "static" wording that are on device in live-common which we did here. this is to avoid having "Ledger Manager" in translation as it's not to be translated.

I think chris want to translate just the "Allow" part, so I think we need to introduce a new "wording".

Reply:
ok. Then this one needs to go as well, https://github.com/LedgerHQ/ledger-live-common/pull/967